sympy/integrals/tests/test_integrals.py

```diff
@@ -402,6 +402,101 @@ def test_issue_13749():
 def test_issue_18133():
     assert integrate(exp(x)/(1 + x)**2, x) == NonElementaryIntegral(exp(x)/(x + 1)**2, x)
 
+def test_issue_21024():
+    x = Symbol('x', real=True, nonzero=True)
+
+    f = log(x)*log(4*x) + log(3*x + exp(2))
+    F = x*log(x)**2 + x*(1 - 2*log(2)) + (-2*x + 2*x*log(2))*log(x) + \
+        (x + exp(2)/6)*log(3*x + exp(2)) + exp(2)*log(3*x + exp(2))/6
+    assert simplify(F - integrate(f,x)) == 0
+
+
+    f = (x + exp(3))/x**2
```

Comment:
Some of these examples are separated by two blank lines and others by one. I think it should use one blank line consistently